less/common/Button.less

```diff
@@ -51,9 +51,10 @@
   white-space: nowrap;
   line-height: 20px;
   padding: 8px 13px;
-  border-radius: @border-radius;
+  border-radius: var(--border-radius);
   .user-select(none);
-  .Button--color(@control-color, @control-bg, 'button');
+  color: var(--button-color);
```

Comment:
Why drop the mixin here?

Reply:
The mixin used to target subelements and add color variations to them, `> .Button-label`, `&:hover`, `&:active` ..etc

with the flexibility of CSS variables, we apply those by default, and we only override the color variables locally by declaring them on a local level.